panel/depends.py

```diff
@@ -32,6 +32,9 @@ def transform_dependency(arg):
         arg = transform(arg)
     return arg
 
+# Alias for backward compatibility
+param_value_if_widget = transform_dependency
```

Comment:
```suggestion
def param_value_if_widget(*args, **kwargs):
    deprecated("1.3", "param_value_if_widget", "transform_dependency")
    return transform_dependency(*args, **kwargs)
```

+ An import on top. 

Note: I just chose a version.